Doc/library/asyncio-task.rst

```diff
@@ -582,9 +582,9 @@ Waiting Primitives
 .. function:: as_completed(aws, \*, loop=None, timeout=None)
 
    Run :ref:`awaitable objects <asyncio-awaitables>` in the *aws*
-   set concurrently.  Return an iterator of :class:`Future` objects.
-   Each Future object returned represents the earliest result
-   from the set of the remaining awaitables.
+   set concurrently. Return an iterator of coroutines.
```

Comment:
```suggestion
   set concurrently.  Return an iterator of coroutines.
```
Some parts of the documentation uses double spaces after `.`. Please keep it so it remains consistent with the surroundings.

Reply:
Didn't know it's on purpose. Fixed :-)